session_1/hello_world/app/views/page/about_me.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>FIRST YEAR ECE STUDENT AT NITK</h1>
```

Comment:
@PRASANNA-416 
Our tests expected you to add some data enclosed in h1 tag, p tag & ol tag. But you are displaying data enclosed only in h1 tags. Although it doesn't matter since you have understood on how to represent data using HTML tags. You can go through the below reference & probably try including your interests in ol tag, display your name in h1 tag & maybe a one line description of yourself in a p tag. 
Reference - (https://www.w3schools.com/tags/tag_ol.asp)